lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
Hi @wtrocki, 
 
First at all thank you for your contributions and effort with it :-)
 
I would like to apologize if I've not been clear enough in the last comments, this way I've removed the last two and created this one with the answer of the question: Why do not just change the code to follow your suggestion to move forward?
 
- I understand that your suggestion won't attend situations as follow, we need to improve it  to make it flexible. 
 
```
 if (!params.json) {
        if (data && data.list && data.list.length > 0) {
          params._table = common.createTable(data.list);
          return cb(undefined, params);
        } else {
          return cb(util.format(i18n._("Not found XXX for the param '%s' into the param '%s'."), params.app, params.env));
        }
      } else {
        return cb(undefined, data.list || data);
      }
```
 
- Remove ```_table``` will break lots of existing tests that we have in the project ;
 
- Test team is working on automation via fhc and this simple improvement/fix will be really helpful to them. Also, the fact that the project is missing this small change is making a big noise and causing unnecessary effort. ( check the JIRAS linked in this one FH 3657, the PR of cmd projects[https://github.com/feedhenry/fh-fhc/pull/341] and the PR into mbaas-art [https://github.com/fheng/mbaas-art/pull/79] )
 
- I think we should first solve the main technical debts which are making all commands follow the same standard and guarantee that they'll be used in the same way in case of the project be used as a dependency. After this main changes, we will have more visibility of the code and we'll be in a better position to make improvements. 
 
- We already have a JIRA to work on after solving the main technical debts, as described above, we'll improve and make ```--json``` parameter as a global thing and we can at this point find out the best approach to improve the output related.  
 
Since you have tested and checked that it works fine and as you said in your comment "Note that this is only a suggestion", could we agree to move forward with this in a simple way and then improve as you suggested after that? Besides this improvement is already planned in another JIRA[FH-3650].
Thank you for your help and contribution so far.